src/components/Resources/Category/CategoriesGrid.js

```diff
@@ -29,6 +29,14 @@ export default function CategoriesGrid({ categories }) {
   return (
     <div>
       <Grid classes={gridStyles} container spacing={spacing}>
+        <Grid item>
+          <CustomCard
+            id="new"
+            title="add category"
+            imageSrc="https://firebasestorage.googleapis.com/v0/b/la-blueprint-ptv.appspot.com/o/resources%2Ficons%2FAddCategory.png?alt=media&token=8fafe6e5-0ad1-4b4b-ac70-9685d286e250"// use code to pull from firebase
```

Comment:
Would it be possible to not hard code this image link? (ie. store it in the firestore as a resource category w/ its own image)